From c5d764f982520e09916f99ef75c06f17f0803714 Mon Sep 17 00:00:00 2001
From: Pedro Alvarez <pedro.alvarez@codethink.co.uk>
Date: Fri, 24 May 2019 14:16:36 +0100
Subject: [PATCH] Use latest Ubuntu 16.04 as base

We also need to stop cleaning '/run/lock', as there is a
'subsys' directory there that we can't cleanup (owned by root)
---
 WORKSPACE                 | 2 +-
 cmd/bb_runner/BUILD.bazel | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index d81afb4..98b725d 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -61,7 +61,7 @@ container_pull(
 
 container_pull(
     name = "rbe_ubuntu16_04_base",
-    digest = "sha256:9bd8ba020af33edb5f11eff0af2f63b3bcb168cd6566d7b27c6685e717787928",
+    tag = "latest",
     registry = "launcher.gcr.io",
     repository = "google/rbe-ubuntu16-04",
 )
diff --git a/cmd/bb_runner/BUILD.bazel b/cmd/bb_runner/BUILD.bazel
index 60907ec..676ccc0 100644
--- a/cmd/bb_runner/BUILD.bazel
+++ b/cmd/bb_runner/BUILD.bazel
@@ -62,7 +62,6 @@ container_image(
     name = "bb_runner_ubuntu16_04_container",
     base = "@rbe_ubuntu16_04_base//image",
     cmd = [
-        "-temp-directory=/run/lock",
         "-temp-directory=/tmp",
         "-temp-directory=/var/tmp",
     ],
-- 
2.20.1

